docs/advanced-features/using-mdx.md

```diff
@@ -85,6 +81,20 @@ Checkout my React component:
 <MyComponent/>
 ```
 
+### Frontmatter
+
+Frontmatter is a YAML like key/value pairing that can be used to store data about a page. `@next/mdx` does **not** support frontmatter by default, though there are many solutions for adding frontmatter to your MDX content, such as [gray-matter](https://github.com/jonschlinkert/gray-matter).
+
+To access page metadata with `@next/mdx`, you can export a meta object from within the `.mdx` file:
+
+```md
+export const meta = {
+author: 'Rich Haines'
```

Comment:
It looks like we didn't get this spacing adjusted - maybe it needs to be `js` instead of `md`?

Reply:
Hmm, that could be it! Rather odd. We should adjust the config for that, as I pulled this out so js makes sense, but it was in the context of an md file, so should format.